src/qml/qml.qrc

```diff
@@ -130,5 +130,6 @@
         <file>imports/Theme/QfMenu.qml</file>
         <file>imports/Theme/QfTimeItem.qml</file>
         <file>imports/Theme/QfCalendarItem.qml</file>
+        <file>MapCanvasPointHandler.qml</file>
```

Comment:
Let's try to keep this file as alphabetically sorted as we can :) In this case it'd also be good to have it stand next to MapCanvas.qml.